app/src/main/res/layout/randomizer_layout.xml

```diff
@@ -9,15 +9,47 @@
         android:layout_height="match_parent"
         android:src="@drawable/smp_dude"
         android:padding="15dip"/>
+    <ImageView
+        android:layout_width="100dp"
+        android:layout_height="700dp"
+        android:src="@drawable/smp_poi"
+        android:padding="20dip"
+        android:layout_marginRight="30dp"
+        android:layout_marginLeft="60dp"
+        android:layout_marginTop="300dp"/>
+    <ImageView
+        android:layout_width="100dp"
+        android:layout_height="700dp"
+        android:src="@drawable/smp_poi"
+        android:padding="20dip"
+        android:layout_marginRight="30dp"
+        android:layout_marginLeft="240dp"
+        android:layout_marginTop="300dp"/>
     <TextView android:id="@+id/button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:gravity="center"
         android:background="@drawable/smp_sign"
-        android:text="Click here for a random poi video!"
+        android:text="Press for a random poi video."
         android:textColor="@drawable/csl_light"
         android:textSize="50dp"
+        android:padding="20dip"
+        android:layout_margin="10dip"
         android:layout_centerInParent="true"
+        android:nestedScrollingEnabled="false"
```

Comment:
I did not add this. I swear. My computer seemed to make tons of changes on its own, which is why I only pushed these 3 files.